Makefile

```diff
@@ -7,6 +7,8 @@ DASHBOARD_CONTAINER_NAME=dashboard-proxy
 DASHBOARD_IMAGE_NAME=jupyter-incubator/$(DASHBOARD_CONTAINER_NAME)
 KG_IMAGE_NAME=jupyter-incubator/kernel-gateway-extras
 KG_CONTAINER_NAME=kernel-gateway
+HTTP_PORT=3000
+HTTPS_PORT=3001
```

Comment:
Should this be `?=`, so it can be overridden? I'm not strong in the shell-fu, but I think that's how it works.